From f7b7a2c937e8da2102328fa9ee17ebac80a3a325 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 19 Jan 2015 13:42:34 -0500
Subject: [PATCH] journalctl: trim --help to fit in 80 columns

Terminals tend to be 80 columns wide by default, and the help
text is only supposed to be a terse reminder anyway.

https://bugzilla.redhat.com/show_bug.cgi?id=1183771
(cherry picked from commit 40f0b71b063da6a36b8d7ec75ef20103abd18243)

Conflicts:
	src/journal/journalctl.c
---
 src/journal/journalctl.c | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/journal/journalctl.c b/src/journal/journalctl.c
index 1ade4ed670..4759c9f759 100644
--- a/src/journal/journalctl.c
+++ b/src/journal/journalctl.c
@@ -193,19 +193,19 @@ static void help(void) {
                "     --system              Show the system journal\n"
                "     --user                Show the user journal for the current user\n"
                "  -M --machine=CONTAINER   Operate on local container\n"
-               "     --since=DATE          Start showing entries on or newer than the specified date\n"
-               "     --until=DATE          Stop showing entries on or newer than the specified date\n"
-               "  -c --cursor=CURSOR       Start showing entries from the specified cursor\n"
-               "     --after-cursor=CURSOR Start showing entries from after the specified cursor\n"
+               "     --since=DATE          Show entries not older than the specified date\n"
+               "     --until=DATE          Show entries not newer than the specified date\n"
+               "  -c --cursor=CURSOR       Show entries starting at the specified cursor\n"
+               "     --after-cursor=CURSOR Show entries after the specified cursor\n"
                "     --show-cursor         Print the cursor after all the entries\n"
-               "  -b --boot[=ID]           Show data only from ID or, if unspecified, the current boot\n"
+               "  -b --boot[=ID]           Show current boot or the specified boot\n"
                "     --list-boots          Show terse information about recorded boots\n"
                "  -k --dmesg               Show kernel message log from the current boot\n"
-               "  -u --unit=UNIT           Show data only from the specified unit\n"
-               "     --user-unit=UNIT      Show data only from the specified user session unit\n"
-               "  -t --identifier=STRING   Show only messages with the specified syslog identifier\n"
-               "  -p --priority=RANGE      Show only messages within the specified priority range\n"
-               "  -e --pager-end           Immediately jump to end of the journal in the pager\n"
+               "  -u --unit=UNIT           Show logs from the specified unit\n"
+               "     --user-unit=UNIT      Show logs from the specified user unit\n"
+               "  -t --identifier=STRING   Show entries with the specified syslog identifier\n"
+               "  -p --priority=RANGE      Show entries with the specified priority\n"
+               "  -e --pager-end           Immediately jump to the end in the pager\n"
                "  -f --follow              Follow the journal\n"
                "  -n --lines[=INTEGER]     Number of journal entries to show\n"
                "     --no-tail             Show all lines, even in follow mode\n"
@@ -226,7 +226,7 @@ static void help(void) {
 #ifdef HAVE_GCRYPT
                "     --interval=TIME       Time interval for changing the FSS sealing key\n"
                "     --verify-key=KEY      Specify FSS verification key\n"
-               "     --force               Force overriding of the FSS key pair with --setup-keys\n"
+               "     --force               Override the FSS key pair with --setup-keys\n"
 #endif
                "\nCommands:\n"
                "  -h --help                Show this help text\n"
@@ -235,7 +235,7 @@ static void help(void) {
                "     --header              Show journal header information\n"
                "     --disk-usage          Show total disk usage of all journal files\n"
                "  -F --field=FIELD         List all values that a specified field takes\n"
-               "     --list-catalog        Show message IDs of all entries in the message catalog\n"
+               "     --list-catalog        Show all message IDs in the catalog\n"
                "     --dump-catalog        Show entries in the message catalog\n"
                "     --update-catalog      Update the message catalog database\n"
                "     --flush               Flush all journal data from /run into /var\n"
